.github/workflows/no-response.yml

```diff
@@ -1,12 +1,7 @@
 name: No Response
 
-# `issues`.`closed`, `issue_comment`.`created`, and `scheduled` event types are required for this Action
-# to work properly.
 on:
-  issues:
-    types: [closed]
-  issue_comment:
-    types: [created]
+  workflow_dispatch:
```

Comment:
Is this correct?

Reply:
what exactly? The removal of the triggers?

Reply:
Yes. This is not the reusable workflow, so I would expect it to keep the original triggers

Reply:
well, apparently the action i use does not need them to function properly ... this is the setting [remove-stale-when-updated](https://github.com/actions/stale?tab=readme-ov-file#remove-stale-when-updated) ... not sure how that works though

Reply:
It probably doesn't work. Here they show how to use the action, but it uses cron https://github.com/actions/stale?tab=readme-ov-file#usage

Reply:
i know ... that's why i have cron in there as well ... i just wanted to run this manually in the start to clear out old issues and prs. If it is important to have an immediate reaction there is no way around doing this via event trigger like issue closed or similar, but IMO we can do a hourly run on this, since the operations should not overflow the rate limit and we dont really need a snappy response on this

Reply:
Ah, nvm the cron was collapsed in the review 🙃